extensions/ql-vscode/src/vscode-tests/minimal-workspace/sourcemap.test.ts

```diff
@@ -0,0 +1,71 @@
+import { fail } from 'assert';
+import { commands, Selection, window, workspace } from 'vscode';
+import * as path from 'path';
+import * as assert from 'assert';
+import { expect } from 'chai';
+import { tmpDir } from '../../helpers';
+import * as fs from 'fs-extra';
+
+export function run() {
```

Comment:
I don't think it is necessary to wrap and export the tests.

Reply:
Isn't that how we ensure that the activation test runs before this new test?

Reply:
Oh...not sure. If this is working for you, can you add a comment here?

Reply:
This is irrelevant now that I moved the test to the `cli-integration` suite